plane/src/database/drone.rs

```diff
@@ -125,11 +129,11 @@ impl<'a> DroneDatabase<'a> {
                 and pool = $3
                 and last_local_time is not null
                 and last_connection_start_time is not null
-            order by drone.id
+            order by drone.id desc
+            limit 100
```

Comment:
nit: since this is hard-coded, maybe add a comment to the function that mentions that the first 100 by ID are returned?

Reply:
Good call